docs/basic/dictionary.md

```diff
@@ -57,6 +57,16 @@ Pengaksesan item menggunakan key yang tidak dikenali akan menghasilkan error.
 
 Sebagai contoh, variabel `profile` di atas jika diakses item dengan key `umur` misalnya (`profile["umur"]`) hasilnya adalah error. 
 
+```python
+profile = {
+    "id": 2,
+    "name": "john wick",
+    "hobbies": ["playing with pencil"],
+    "is_female": False,
+}
+
+print("age:", profile["age"])
+# output ➜ KeyError: 'age'
```

Comment:
let us not use `output` because that word is better used when the app generates stdout output.

As this statement you use here generates on stderr, I prefer to use `# KeyError: 'age'` instead of `# output ➜ KeyError: 'age'`

Reply:
Oke, thanks i will change and remove the output

Reply:
Also, i want change the sentence " misalnya (`profile["umur"]`) " to " misalnya (`profile["age"]`) "